apps/pie-docs/src/components/modal/overview/overview.md

```diff
@@ -0,0 +1,424 @@
+    width: 336
+} %}
+
+### Modifiers
+
+#### Footer
+
+As well as the standard Modal footer modifiers, at a narrow size you have the option to have full width Buttons that are stacked.
+
+{% contentLayout %}
+  {% contentPageImage {
+      src: "../../../assets/img/components/modal/overview/narrow-modifier-footer-dual-action.png",
+      alt: "A modal with a single primary button in its footer on a mobile screen.",
+      width: 336
+  } %}
+  {% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/narrow-modifier-footer-single-action.png",
+    alt: "A modal with a single primary button in its footer on a mobile screen.",
+    width: 288
+  } %}
+{% endcontentLayout %}
+
+---
+
+## Behaviours
+
+### Loading
+
+In order to keep the size consistent when the content inside the Modal is loading, we have set a fixed height of 360px while the loading spinner is visible.
+
+{% contentPageImage {
+  src: "../../../assets/img/components/modal/overview/behaviour-loading.png",
+  alt: "Loading behaviour of a modal.",
+  width: 600
+} %}
+
+---
+
+## Layout
+
+### Position
+
+{% notification {
+  type: "information",
+  message: "Only applicable to Modals that aren’t full screen."
+} %}
+
+#### Center
+
+The Modal is positioned at the center of the screen by default.
+
+{% contentPageImage {
+  src: "../../../assets/img/components/modal/overview/layout-position-center.png",
+  alt: "A modal that is positioned in the middle of the screen.",
+  shouldShowPadding: true
+} %}
+
+#### Top
+
+The Modal is positioned at the center of the screen by default.
+
+{% contentPageImage {
+  src: "../../../assets/img/components/modal/overview/layout-position-top.png",
+  alt: "A modal that is positioned in the middle of the screen.",
+  shouldShowPadding: true
+} %}
+
+---
+
+## Overflow
+
+### Title
+
+When the title exceeds the available width, it wraps onto a new line.
+
+{% contentPageImage {
+  src: "../../../assets/img/components/modal/overview/overflow-title.png",
+  alt: "A modal with a long title.",
+  width: 600
+} %}
+
+### Body content
+
+When the Modal’s content is longer than the available Modal height, then the body section should scroll vertically with the header, and a footer fixed in place. Modal content should never scroll horizontally; instead, use a larger size modal.
+
+#### Footer pinned = true
+
+If the actions are required to be sticky, they are pinned to the bottom of the Modal container whilst the user scrolls to indicate there is more content below until.
+
+{% contentPageImage {
+  src: "../../../assets/img/components/modal/overview/overflow-body-content-footer-pinned.png",
+  alt: "A modal with a pinned footer.",
+  width: 400
+} %}
+
+#### Footer pinned = false
+
+If the actions aren’t required to be sticky and placed at the bottom of the content, and the passive footer is pinned to the bottom of the Modal container whilst the user scrolls to indicate there is more content below.
```

Comment:
I think this parapraph could do with rewording

Reply:
Should it just be

'If the actions aren’t required to be sticky and are placed at the bottom of the content, the passive footer is pinned to the bottom of the Modal container whilst the user scrolls to indicate there is more content below.'?

Reply:
"indicates the presence of additional content below"?